app/Models/V2/Projects/Project.php

```diff
@@ -312,6 +312,11 @@ public function users(): BelongsToMany
         return $this->belongsToMany(User::class, 'v2_project_users')->withPivot(['status', 'is_monitoring']);
     }
 
+    public function managers(): BelongsToMany
+    {
+        return $this->belongsTomany(User::class, 'v2_project_users')->wherePivot('is_managing', true);
```

Comment:
is this a typo or this is intended? `belongsTomany`

Reply:
Oh wild, today I learned that PHP function names are case insensitive. What an "interesting" language 😂 . Fixing now.